packages/charts/src/vaadin-chart-mixin.js

```diff
@@ -0,0 +1,1701 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ *
+ * See https://vaadin.com/commercial-license-and-service-terms for the full
+ * license.
+ */
+import 'highcharts/es-modules/masters/highstock.src.js';
+import 'highcharts/es-modules/masters/modules/accessibility.src.js';
+import 'highcharts/es-modules/masters/modules/annotations.src.js';
+import 'highcharts/es-modules/masters/highcharts-more.src.js';
+import 'highcharts/es-modules/masters/highcharts-3d.src.js';
+import 'highcharts/es-modules/masters/modules/data.src.js';
+import 'highcharts/es-modules/masters/modules/drilldown.src.js';
+import 'highcharts/es-modules/masters/modules/exporting.src.js';
+import 'highcharts/es-modules/masters/modules/funnel.src.js';
+import 'highcharts/es-modules/masters/modules/heatmap.src.js';
+import 'highcharts/es-modules/masters/modules/solid-gauge.src.js';
+import 'highcharts/es-modules/masters/modules/treemap.src.js';
+import 'highcharts/es-modules/masters/modules/no-data-to-display.src.js';
+import 'highcharts/es-modules/masters/modules/sankey.src.js';
+import 'highcharts/es-modules/masters/modules/timeline.src.js';
+import 'highcharts/es-modules/masters/modules/organization.src.js';
+import 'highcharts/es-modules/masters/modules/xrange.src.js';
+import 'highcharts/es-modules/masters/modules/bullet.src.js';
+import 'highcharts/es-modules/masters/modules/gantt.src.js';
+import 'highcharts/es-modules/masters/modules/draggable-points.src.js';
+import { FlattenedNodesObserver } from '@polymer/polymer/lib/utils/flattened-nodes-observer.js';
+import { beforeNextRender } from '@polymer/polymer/lib/utils/render-status.js';
+import Pointer from 'highcharts/es-modules/Core/Pointer.js';
+import Highcharts from 'highcharts/es-modules/masters/highstock.src.js';
+import { get } from '@vaadin/component-base/src/path-utils.js';
+import { ResizeMixin } from '@vaadin/component-base/src/resize-mixin.js';
+import { inflateFunctions } from './helpers.js';
+
+/** @private */
+export function deepMerge(target, source) {
```

Comment:
We could move this to `helpers.js` so that we can import it from there also in `vaadin-chart-series-mixin`.

Reply:
Done